src/content/content.js

```
@@ -65,71 +84,122 @@ injectJs(chrome.runtime.getURL("injected.js"));
+      break;
+
+    case "unloadHeaderHTML":
+      const headerHTMLCode =
+        document.getElementsByName("header")[0].childNodes[0] || "";
+      logDebug("Unloading header HTML code");
+      sendResponse({ code: headerHTMLCode });
+      break;
+
+    case "unloadFooterHTML":
+      const footerHTMLCode =
+        document.getElementsByName("footer")[0].childNodes[0] || "";
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)